saleor/core/logging.py

```diff
@@ -47,3 +49,20 @@ def add_fields(self, log_record, record, message_dict):
             }
         )
         super().add_fields(log_record, record, message_dict)
+
+
+class DisableBadRequestGraphQLLog(logging.Filter):
+    """Filter to disable logging of "Bad Request" messages in the GraphQL endpoint.
+
+    This is useful to prevent cluttering the logs with expected "Bad Request" errors
+    that occur when clients send invalid GraphQL queries.
+    """
+
+    def filter(self, record):
+        api_path = reverse("api")
```

Comment:
I think this one can be costly to run. Are there any `Bad Request` warnings or even any other warnings from the same logger that we wish to keep? If not, we could silence that entire logger in production.

Reply:
So maybe we should raise the logger's level in production to ERROR. Regarding warnings, we're getting events such as:
- Bad Request
- Too Many Requests
- Unauthorized
- Not Found

These warnings depend on the client's behavior and do not signal issues with the backend itself.

Reply:
Does it mean there will be no info/warning logs, ever?